src/app/contextBroker/contextBroker.cpp

```diff
@@ -271,6 +274,7 @@ bool            statNotifQueue;
 #define PIDPATH             _i "/tmp/contextBroker.pid"
 #define IP_ALL              _i "0.0.0.0"
 #define LOCALHOST           _i "localhost"
+#define MAX_PERIOD          (3600 * 24 * 31)  // Once a month ...
```

Comment:
What about ONE_MOTH_PERIOD? ONE_MONTH could be misleaded with other things meaning one month (e.g. a string).